backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/project/workOrder/WorkOrderProjectRestController_StepDef.java

```diff
@@ -232,16 +229,20 @@ private void validateJsonWorkOrderObjectsUnderTestResponseList(
 		final List<JsonWorkOrderObjectsUnderTestResponse> objectsUnderTest = jsonWorkOrderProjectResponse.getObjectsUnderTest();
 		final List<JsonWorkOrderObjectsUnderTestResponse> expectedObjectsUnderTest = expectedJsonWorkOrderProjectResponse.getObjectsUnderTest();
 
-		assertThat(objectsUnderTest).isNotNull();
-		assertThat(expectedObjectsUnderTest).isNotNull();
+		final SoftAssertions softly = new SoftAssertions();
+
+		softly.assertThat(objectsUnderTest).isNotNull();
```

Comment:
same question..i'm asking because if i look at the SoftAssertions javadoc, they make heavy use of `.as(....)` and i can't imagine how SoftAssertions would give any benefit without that..but maybe they do some sort of introspection..